__tests__/core_kit.spec.coffee

```diff
@@ -19,6 +19,11 @@ test 'Making a request with JSON response', (done) ->
 
     return
 
+test 'Making a request with JSON response, promises async/await style', ->
+    data = await SGN.CoreKit.request url: '/v2/catalogs'
+    expect(data).toBeDefined()
```

Comment:
I think we should have spacing between function calls and variable declarations. I don't know what the Coffee rule is but eslint rule is: `"keyword-spacing": ["error", { "before": true, "after": true }],`.